gammapy/maps/hpxsparse.py

```diff
@@ -24,16 +25,23 @@ class HpxSparseMap(HpxMap):
         HEALPIX geometry object.
     data : `~numpy.ndarray`
         HEALPIX data array.
+    meta : `~collections.OrderedDict`
+        Dictionary to store meta data.
     """
 
-    def __init__(self, geom, data=None, dtype='float32'):
+    def __init__(self, geom, data=None, dtype='float32', meta=None):
         if data is None:
             shape = tuple([np.max(geom.npix)] + [ax.nbin for ax in geom.axes])
             data = SparseArray(shape[::-1], dtype=dtype)
         elif isinstance(data, np.ndarray):
             data = SparseArray.from_array(data)
 
-        super(HpxSparseMap, self).__init__(geom, data)
+        if meta is None:
```

Comment:
Why are you initializing the meta dictionary here?  Doesn't it already get initialized in the `Map` constructor?